src/govuk/components/notification-banner/notification-banner.yaml

```diff
@@ -0,0 +1,134 @@
+params:
+- name: title
+  type: string
+  required: false
+  description: Custom title text. Defaults to 'Important' ('Success' for success banner and 'Error' for error banner). If `titleHtml` is supplied, this is ignored.
+- name: titleHtml
+  type: string
+  required: false
+  description: Custom title HTML. Defaults to 'Important' ('Success' for success banner and 'Error' for error banner).
+- name: titleHeadingLevel
+  type: string
+  required: false
+  description: Heading level, from 1 to 6. Default is `2`.
+- name: content
+  type: string
+  required: true
+  description: Content for the notification banner. If `contentHtml` is supplied, this is ignored.
+- name: contentHtml
+  type: string
+  required: true
+  description: HTML content for the notification banner.
+- name: type
+  type: string
+  required: false
+  description: Sets classes and attributes to create a type of notification banner. Accepts `success` or `alert`.
```

Comment:
I know setting this to 'notice' (or similar) wouldn't actually do anything, but it seems a little strange to only list the 2 'success' or 'alert' types here, when really there's 3 'types' and one of them just happens to be the default.

Reply:
@36degrees I know what you mean. If this said 
> Accepts `success` or `alert`. Defaults to `notification`.

would that make more sense? (I could check with Chris if he's got thoughts on what the default `type` should be called.

Reply:
I've improved the documentation here but I'm also going to have a look at it with Eoin.

Reply:
The new documentation is really good, but focuses only on the `role` and `tabindex` attributes and the focus behaviour. The `type` also affects the style of the banner, and the default title.

Reply:
I'll mark this as resolved though as I think that's separate.